src/test/java/app/bpartners/api/service/WMS/imageSource/WmsImageSourceFacadeIT.java

```diff
@@ -85,8 +85,7 @@ void downloadImage_cascade_on_server_error_ok() {
 
     File actual = subject.downloadImage(GEOSERVER_LAYER_AREA_PICTURE);
 
-    verify(geoserverImageSourceMock, times(2)).downloadImage(any());
-    verify(openStreetMapImageSourceMock, times(1)).downloadImage(any());
+    verify(geoserverImageSourceMock, times(1)).downloadImage(any());
```

Comment:
not enough.

This test should cover the following case : 
an AreaPicture download request is sent.
We try to find an image with the given geoserver (it should be a server other than IGN for this test)
 -> it fails.
 
 Then on failure, our program will contact geoserver with IGN layer -> success.
 
 So you should check that geoserver has been called exactly twice (with one failure and one success)